README.md

```diff
@@ -34,45 +34,56 @@ Basic logging will be displayed by default, including the number of high-level C
 objects loaded and the name of the output files.
 ```
 > python3 bin/ons_csv_to_ctb_json_main.py -i test/testdata/ -g test/testdata/geography/geography.csv -o ctb_metadata_files/
-t=2022-04-28 15:21:06,357 lvl=INFO msg=Reading test/testdata/geography/geography.csv: found Welsh labels for unknown classification: OTHER
-t=2022-04-28 15:21:06,357 lvl=INFO msg=Dropped non public classification: CLASS_PRIV
-t=2022-04-28 15:21:06,357 lvl=INFO msg=Loaded metadata for 5 Cantabular variables
-t=2022-04-28 15:21:06,358 lvl=INFO msg=Loaded metadata for 3 Cantabular datasets
-t=2022-04-28 15:21:06,359 lvl=INFO msg=Written dataset metadata file to: ctb_metadata_files/cantabm_v9-3-0_unknown-metadata-version_dataset-md_20220428-1.json
-t=2022-04-28 15:21:06,360 lvl=INFO msg=Dropped non public ONS Dataset: DS_PRIV
-t=2022-04-28 15:21:06,360 lvl=INFO msg=Loaded metadata for 4 Cantabular tables
-t=2022-04-28 15:21:06,360 lvl=INFO msg=Written table metadata file to: ctb_metadata_files/cantabm_v9-3-0_unknown-metadata-version_tables-md_20220428-1.json
-t=2022-04-28 15:21:06,360 lvl=INFO msg=Loaded service metadata
-t=2022-04-28 15:21:06,360 lvl=INFO msg=Written service metadata file to: ctb_metadata_files/cantabm_v9-3-0_unknown-metadata-version_service-md_20220428-1.json
+t=2022-05-09 21:26:50,348 lvl=INFO msg=ons_csv_to_ctb_json_main.py version 1.1.alpha
```

Comment:
Introduction text still states: "It is compatible with version `1.1` of the metadata schema and version `9.3.0` of `cantabular-metadata`."

Reply:
OK, will update.